### PR TITLE
Update default Storage rules to contain example cross-srevice-rules

### PR DESCRIPTION
**Pull request by sam-gc**
To help visibility for this feature

### CODE REVIEW

File: templates/init/storage/storage.rules [hunk 1]
```diff
@@ -1,4 +1,8 @@
 rules_version = '2';
+
+// Craft rules based on data in your Firestore database
```
Comment (joehan) [inline templates/init/storage/storage.rules:3]: Consider including a relevant docs links here?

**Review by joehan — APPROVED** [review]
Good idea!

File: templates/init/storage/storage.rules (hunk 1 above)
Comment (sam-gc) [reply templates/init/storage/storage.rules:3]: Ah good point. This copy was approved with PM so let me double check about the link and get back to this

Comment (yuchenshi) [reply templates/init/storage/storage.rules:3]: Please move this into a `match` block otherwise it won't work when you uncomment it as-is

Comment (sam-gc) [reply templates/init/storage/storage.rules:3]: It's meant not to be uncommentable like that (to prevent accidentally opening things up)

Comment (sam-gc) [reply templates/init/storage/storage.rules:3]: I'm going to go ahead and merge this as-is. These rules match the Console and in Console there's a banner already with a link to the docs.

**Pull request merged**